examples/geospatial/README.txt

```diff
@@ -1,2 +1,58 @@
 Geospatial
 ----------
+
+The following geospatial examples showcase different ways of performing
+network analyses using packages within the geospatial Python ecosystem.
+Example spatial files are stored directly in this directory.
+
+Dependencies
+~~~~~~~~~~~~
+
+`GeoPandas <https://geopandas.readthedocs.io/>`__ provides
+interoperability between geospatial formats and storage mechanisms
+(e.g., databases) and Pandas data frames for tabular-oriented processing
+of spatial data, as well as a wide array of supporting functionality
+including spatial indices, spatial predicates (e.g., test if geometries
+intersect each other), spatial operations (e.g., the area of overlap
+between intersecting polygons), and more.
+
+`PySAL <https://pysal.org/>`__ provides a rich suite of spatial analysis
+algorithms. From a network analysis context, `spatial
+weights <https://pysal.org/libpysal/api.html#spatial-weights>`__
+provide… (Levi please add more here).
+
+See the following examples that use PySAL:
+
+* ``examples/geospatial/plot_delaunay.py``
+* ... (depends on split between PySAL and Momepy examples)
+
+`momepy <http://docs.momepy.org/en/stable/>`__ builds on top of
+GeoPandas and PySAL to provide a suite of algorithms focused on urban
+morphology. From a network analysis context, momepy enables you to
+convert your line geometry to ``networkX.MultiGraph`` and back to 
+``GeoDataFrames`` and apply a range of analytical functions aiming at 
+morphological description of (street) network configurations.
+
+See the following examples that use momepy:
+
+* ...
+
```

Comment:
```suggestion

`OSMnx <https://osmnx.readthedocs.io/>`__ provides a set of tools to retrieve, model, project, analyze, and visualize OpenStreetMap street networks (and any other networked infrastructure) as NetworkX MultiDiGraph objects, and convert these MultiDiGraphs to/from Geopandas GeoDataFrames. It can automatically add node/edge attributes for: elevation and grade (using the Google Maps Elevation API), edge travel speed, edge traversal time, and edge bearing. It can also retrieve any other spatial data from OSM (such as building footprints, public parks, schools, transit stops, etc) as Geopandas GeoDataFrames.

See the following examples that use OSMnx:

* ``examples/geospatial/plot_osmnx.py``

```